src/xcode/ENA/ENA/Source/Models/Home/HomeTestResultLoadingCellConfigurator.swift

```diff
@@ -21,15 +21,19 @@ import Foundation
 
 class HomeTestResultLoadingCellConfigurator: CollectionViewCellConfigurator {
 
-	let identifier = UUID()
-
 	func configure(cell: HomeTestResultLoadingCell) {
 		cell.setupCell()
 		cell.title.text = AppStrings.Home.resultCardLoadingTitle
 		cell.body.text = AppStrings.Home.resultCardLoadingBody
 		cell.button.isEnabled = false
 		cell.button.setTitle(AppStrings.Home.resultCardShowResultButton, for: .disabled)
-		return
 	}
 
+	// MARK: Hashable
+
+	func hash(into hasher: inout Swift.Hasher) { }
```

Comment:
same as above

Reply:
see my answer for `HomeFindingPositiveRiskCellConfigurator`